docs/README.md

```diff
@@ -201,6 +201,30 @@ versions:
   for Linux
 - [`nvm-windows`](https://github.com/coreybutler/nvm-windows) - Works with native Windows
 
+##### Install Git
+Booster will initialize a Git repository when you create a new project, so it is required that you have it already installed it in your system.
+
+###### Ubuntu
+
+```shell
+sudo apt install git-all
+```
+
+###### macOS
+
+```shell
+brew install git
+```
+
+###### Windows
+
+```shell
+choco install git
+```
+
+After installing git in your machine, you need to take a look in the global variables `user.name` and `user.email`, that should be properly configured globally.
+you can can check this page to [config your global variables](https://git-scm.com/docs/git-config)
+
```

Comment:
Awesome @jvmazagao , just a couple more suggestions

Can you write here the two steps to configure this?

Also can you please change `global variables` to `Git configuration variables`?

Reply:
I will add the steps